controllers/druid/handler.go

```diff
@@ -1190,14 +1190,7 @@ func getRollingUpdateStrategy(nodeSpec *v1alpha1.DruidNodeSpec) *appsv1.RollingU
 			},
 		}
 	}
-	return &appsv1.RollingUpdateDeployment{
-		MaxUnavailable: &intstr.IntOrString{
-			IntVal: int32(25),
-		},
-		MaxSurge: &intstr.IntOrString{
-			IntVal: int32(25),
-		},
-	}
+	return &appsv1.RollingUpdateDeployment{}
 
 }
```

Comment:
https://github.com/datainfrahq/druid-operator/blob/master/controllers/druid/handler.go#L1183

1. If no value is specified it is already kept to 25.
2. If specified value return what value user wants.

IMO this PR isn't needed

Reply:
It's an int 25 and not a percentage of replicas in the deployment.

on a rollingUpdate theses default parameters will delete 25 existing pods and create 25 pods in parallel (less if we have less than 25 replicas)

it could lead some understood and missconfiguration (on cloud setup for example)

Reply:
oh i see, i get what you are saying. Make sense, we can default it to nil, and deployment controller takes care of it.